osgp/platform/osgp-adapter-ws-smartmetering/src/main/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/mapping/EventConverter.java

```diff
@@ -68,6 +73,12 @@ public Event convertFrom(final org.opensmartgridplatform.adapter.ws.schema.smart
         final org.opensmartgridplatform.domain.core.valueobjects.smartmetering.EventLogCategory eventLogCategory =
                 org.opensmartgridplatform.domain.core.valueobjects.smartmetering.EventLogCategory.fromValue(
                 source.getEventLogCategory().value());
-        return new Event(timestamp, eventCode, source.getEventCounter(), eventLogCategory);
+        DateTime startTime = null;
```

Comment:
you could move this to a separate method and do the call in the constructor call to make the code more compact.